src/theory/sets/theory_sets_private.cpp

```diff
@@ -2285,25 +1251,10 @@ Theory::PPAssertStatus TheorySetsPrivate::ppAssert(TNode in, SubstitutionMap& ou
       }
     }
   }
-
-  if( status==Theory::PP_ASSERT_STATUS_SOLVED ){
-    Trace("sets-var-elim") << "Sets : ppAssert variable eliminated : " << in << ", var = " << var << std::endl;
-    /*
-    if( var.getType().isSet() ){
-      //we must ensure that subs is included in the universe set
-      d_var_elim[var] = true;
-    } 
-    */
-    if( options::setsExt() ){
-      Assert( !var.getType().isSet() ); 
```

Comment:
Since this assertion is removed, the local variable `var` is no longer needed, and it can be removed (as CLion says).